doc/next/_sources/campaign-overview.rst.txt

```diff
@@ -0,0 +1,86 @@
+
+..  include:: /campaign-restrictions.rst
+
+
+To give you an idea, it is possible to pause a campaign, resulting in all ads in the campaign to be taken offline. When a campaign reaches one of its :ref:`campaign_budgets`, all ads in that campaign will be taken offline. When budget becomes available again, all ads in the campaign will come
+online again.
+
+.. _components:
+
+Components
+----------
+
+A campaign consists of several components:
+
+========================= ================================================
+Component                 Short description
+========================= ================================================
+:ref:`campaign_id`        The unique reference to the campaign
+:ref:`campaign_title`     User-provided title
+:ref:`campaign_vendorid`  Unique reference for the seller to the campaign
+:ref:`campaign_status`    Status of the campaign
+:ref:`campaign_budgets`   Budgets of this campaign
+:ref:`campaign_targeting` Targeting options of this campaign
+========================= ================================================
+
+
+.. _campaign_id:
+
+Campaign ID
+"""""""""""
+Unique reference to the Campaign, needs to be omitted in a POST action. Generated by us,
+and needed for certain GET/PUT actions to modify the campaign and its properties.
+
+
+.. _campaign_title:
+
+Campaign title
+""""""""""""""
+The title of the campaign, provided by you. This makes it easier for you to distingish between your campaigns. Your campaign titles are not visible to your ad viewers.
+
+.. _campaign_vendorid:
+
+Campaign vendor ID
+""""""""""""""""""
+A per-seller unique identifier to identify the campaign. This is provided by you.
+Can either be set when creating a campaign or when updating an
+existing campaign. However, once set, it can no longer be modified. When fetching an
+existing campaign which does not have a **vendorId**, the field is omitted.
+Creating a second campaign with a vendor ID already used by the seller is not allowed.
+Since this also functions as a unique identifier (for the seller) it is possible to fetch the
+campaign by vendor ID using :ref:`get_campaign_vendorid`.
+
+.. _campaign_status:
+
+Campaign status
+"""""""""""""""
+
+The status the campaign currently has. It can be any value from the list in :ref:`campaign_status_overview`. You can set only one
+of the user-controlled states *ACTIVE*, *PAUSED* or *DELETED*. *DELETED* is a final state.
+
+.. _campaign_budgets:
+
+Campaign budgets
+""""""""""""""""
+
+Campaigns have multiple types of budgets, see :ref:`campaign_budgets_overview`. Budgets are seen as limiting factors, meaning that, as soon
+as one runs out, all ads of this campaign will be taken offline until budget(s) becomes available. Campaign budgets **must be defined**, even if they are set to *unlimited*.
+Absence of campaign budgets means that the ads will not be live.
```

Comment:
When campaign budget(s) are not set, the ads for this campaign will *not* be live.